pkg/server/status.go

```diff
@@ -2832,6 +2917,90 @@ func (s *statusServer) iterateNodes(
 	return resultErr
 }
 
+// iterateNodes iterates nodeFn over all non-removed nodes concurrently.
+// It then calls nodeResponse for every valid result of nodeFn, and
+// nodeError on every error result.
+func (s *statusServer) callNodes(
```

Comment:
`callNodes` is identical to `dialNodes` function with only difference that nodes to dial are provided as argument instead of getting them from `s.serverIterator.getAllNodes(ctx)` (as commented out below).
It allows to dial specific nodes (known to gossip client in our case).